packages/forms/src/UIForm/fields/CheckBox/SimpleCheckBox.component.js

```diff
@@ -17,7 +17,7 @@ export default function SimpleCheckBox({
 	const { autoFocus } = schema;
 	return (
 		<div className={classnames('checkbox', { disabled })}>
-			<label>
+			<label data-feature={schema['data-feature']}>
```

Comment:
Don't know much yet schema, but are you sure that you have property `data-feature` in schema?
Maybe a 
```
import { get } from 'lodash';
get(schema, 'data-feature', '');
```
or equivalent to prevent Error ?

Reply:
Thank you @vbocquet for your comment but, no, because if `data-feature` is empty, it will generate en extra empty attribute. `schema` can't be null in that case :)

Reply:
Perfect, thanks for information 👍